docs/xls-47d-price-oracles/ledger-object-types/oracle.md

```diff
@@ -0,0 +1,92 @@
+---
+html: oracle.html
+parent: ledger-object-types.html
+blurb: The definition and details of an oracle object.
+labels:
+  - Oracle
+status: not_enabled
+---
+# Oracle
+[[Source]](https://github.com/XRPLF/rippled/blob/bf6f5294a9b83653888600e78da8650896e9d393/src/ripple/protocol/impl/LedgerFormats.cpp#L343-L356 "Source")
+<!-- TODO: Update source link to merged version when available -->
+
+<embed src="/snippets/_price-oracles-disclaimer.md" />
+
+An `Oracle` ledger entry holds references to, or data associated with, a single price oracle.
+
+
+## Example Oracle JSON
+
+**TODO: Need sample.**
+
+```json
+{
+  "LedgerEntryType": "Oracle",
+  "Owner": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+  "Provider": "70726F7669646572",
+  "AssetClass": "63757272656E6379",
+  "PriceDataSeries": [
+    {
+      "PriceData": {
+        "BaseAsset": "XRP",
+        "QuoteAsset": "USD",
+        "AssetPrice": 74,
+        "Scale": 2,
+      }
+    },
+  ],
+  "LastUpdateTime": 743609414,
+  "PreviousTxnID": "C53ECF838647FA5A4C780377025FEC7999AB4182590510CA461444B207AB74A9",
+  "PreviousTxnLgrSeq": 56865244
+}
+```
+
+
+## Oracle Fields
+
+| Field               | JSON Type | Internal Type | Required? | Description |
+|---------------------|-----------|---------------|-----------|-------------|
+| `Owner`             | String    | AccountID     | Yes       | The XRPL account with update and delete privileges for the oracle. It's recommended to set up [multi-signing](https://xrpl.org/set-up-multi-signing.html) on this account. |
+| `Provider`          | String    | Blob          | Yes       | An arbitrary value that identifies an oracle provider, such as Chainlink, Band, or DIA. This field is a string, up to 256 ASCII hex encoded characters (0x20-0x7E). |
+| `PriceDataSeries`   | Array     | Array         | Yes       | An array of up to 10 `PriceData` objects, each representing the price information for a token pair. More than five `PriceData` objects require two owner reserves. |
+| `LastUpdateTime`    | Number    | UInt32        | Yes       | The time the data was last updated, represented in the [ripple epoch time](https://xrpl.org/basic-data-types.html#specifying-time). |
+| `URI`               | String    | Blob          | No        | An optional Universal Resource Identifier to reference price data off-chain. This field is limited to 256 bytes. |
+| `AssetClass`        | String    | Blob          | Yes       | Describes the type of asset, such as "currency", "commodity", or "index". This field is a string, up to 16 ASCII hex encoded characters (0x20-0x7E). |
+| `OwnerNode`         | ???       | ???           | Yes       | ??? |
```

Comment:
OwnerNode (see https://xrpl.org/paychannel.html): A hint indicating which page of the source address's owner directory links to this entry, in case the directory consists of multiple pages.

Reply:
Added field details.